UndertaleModLib/Util/BufferBinaryReader.cs

```diff
@@ -296,13 +285,12 @@ public string ReadGMString()
                     chunkBuffer.Read(buf);
                 res = encoding.GetString(buf);
             }
-            
-#if DEBUG
+
             if (ReadByte() != 0)
+            {
                 throw new IOException("String not null terminated!");
-#else
-            Position++;
```

Comment:
The diff seems to suggest that something important was removed here?

Reply:
That's removing the old release code, it now always goes through the ReadByte() path instead.

Reply:
ah right.